src/styles/blocks/calendar.scss

```
@@ -0,0 +1,25 @@
+@import "./variables/variables";
+@import "./utils/mixins";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  width: $calendar-width;
+  padding: 20px 0;
+
+  &__day {
+    @include cell($cell-size);
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
```

Create variable for transition duration
```suggestion
    transition: background-color $transitionDuration, transform $transitionDuration;
```